Models/Base/MetaData/POCO/Axes.cs

```diff
@@ -0,0 +1,11 @@
+using System.Text.Json.Serialization;
+
+namespace Backend.Models.Base.MetaData.POCO
+{
+    public class Axes
+    {
+        [JsonPropertyName("x")] public X X { get; set; }
+
+        [JsonPropertyName("y")] public Y Y { get; set; }
```

Comment:
No need to have two separate classes X and Y, could rather have one class called Axis.
```csharp
[JsonPropertyName("x")] public Axis X { get; set; }
[JsonPropertyName("y")] public Axis Y { get; set; }
```